Qausim/react-weather-app
# Connect App with API

#### What does this PR do?
Connects the app to the APIs for dynamic data

#### Description of Task to be completed?
Send request to the APIs when user inputs a location

#### How should this be manually tested?
Clone this repo, cd into it and 
- Run `npm start`, open `localhost:3000` in a browser, input a valid location and submit. Fresh data should be displayed

#### Any background context you want to provide?
Dynamic weather data from APIs
